packaging/installer/README.md

```diff
@@ -3,7 +3,6 @@ title: "Installation guide"
 custom_edit_url: https://github.com/netdata/netdata/edit/master/packaging/installer/README.md
 -->
 
-import { Install, InstallBox } from '../../../src/components/Install/'
```

Comment:
That is probably used by https://learn.netdata.cloud/, but not sure 🤷‍♂️ 

Why do you think we can delete it?
---

Learn components: https://github.com/netdata/learn/tree/master/src/components

Other such imports:

```cmd
[ilyam@pc netdata]$ grep "import {" --include \*.md --include \*.mdx -R *
docs/get-started.mdx:import { OneLineInstall } from '../src/components/OneLineInstall/'
docs/get-started.mdx:import { Install, InstallBox } from '../src/components/Install/'
docs/store/change-metrics-storage.md:import { Calculator } from '../../src/components/agent/dbCalc/'
packaging/installer/README.md:import { Install, InstallBox } from '../../../src/components/Install/'
```

Reply:
I wasn't aware of it, You probably right.

Reply:
Ok, i guess that InstallBox thing is used to create https://learn.netdata.cloud/docs/agent/packaging/installer#alternative-methods

Reply:
I see, Reverted. thank you